lib/Dist/Zilla.pm

```diff
@@ -799,6 +799,29 @@ sub stash_named {
   return $self->_global_stashes->{ $name };
 }
 
+=method _call_plugin_method
```

Comment:
Private methods aren't normally documented, because otherwise people will use them, and private methods are... private :)